### PR TITLE
Assignment5 @PlantBrush

### PR DESCRIPTION
**Pull request by NazariyPogo**
Partner @PlantBrush
URL: https://lemon-ground-09e5fcf1e.1.azurestaticapps.net
Endpoint: https://wordle-web-api.azurewebsites.net/swagger

Logins:
Admin             'Admin@intellitect.com' 'P@ssw0rd123'
Old Master      'Master@intellitect.com' 'P@ssw0rd123'
Young Master  'Young@intellitect.com' 'P@ssw0rd123'
Normie            'User@intellitect.com' 'P@ssw0rd123'

### CODE REVIEW

**Comment by dcox2005** [comment]
api link says it's not found.

**Comment by NazariyPogo** [comment]
> api link says it's not found.

whoops, added /swagger so it now leads directly there

**Comment by JessicaH46** [comment]
Looks good so far, I did notice that the search was not working when I tested it. Other than that good job so far!

File: wordle-web/pages/wordlist.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        .get('/Word/GetFilteredList', {
+          params: {
+            pageNum: this.page,
+            pageSize: this.wordPerPage,
+            filter: this.search,
+          },
+        })
+        .then((res) => (this.words = res.data))
+        .then(this.getListSize)
+    }
```
Comment (A-Tanner) [inline wordle-web/pages/wordlist.vue:126]: Minor ux issue when you search for words. The page is set to be the current page after retrieving filtered words, which can result in a query appearing to deliver no results if you are on, say, page 4 but the query only produces two pages of results. Would recommend setting the page number to 1 after updating the list.

**Review by A-Tanner — COMMENTED** [review]
Really solid work! The back end seems well made and the front end is pleasant + functional. I had to dig pretty hard for a bug and could only find one!

File: wordle-web/pages/wordlist.vue [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        </v-row>
+        <v-row>
+          <v-col :cols="9">
+            <v-text-field v-model="newWord" label="New Word"> </v-text-field>
+          </v-col>
+          <v-col>
+            <v-btn @click="addWord"> Add Word </v-btn>
+          </v-col>
+        </v-row>
+        <v-simple-table>
```
Comment (mmwoodfo) [inline wordle-web/pages/wordlist.vue:22]: This doesn't seem to be viewable when not logged in

**Review by meghanmae — COMMENTED** [review]
Thanks for the logins in your PR comment. :)

File: wordle-web/pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <v-card>
+          <v-card-title>Login</v-card-title>
+          <v-row>
+            <v-col cols="12" md="4">
+              <v-text-field v-model="email" label="Email" required>
+              </v-text-field>
+            </v-col>
+
+            <v-col cols="12" md="4">
+              <v-text-field
```
Comment (mmwoodfo) [inline wordle-web/pages/login.vue:14]: This should probably be of type password as well

**Pull request closed without merging**